plugins/radar-android-google-places/src/main/java/org.radarbase.passive.google.places/GooglePlacesService.kt

```diff
@@ -101,10 +108,22 @@ class GooglePlacesService: SourceService<GooglePlacesState>() {
         }
     }
 
+    @Synchronized
+    fun createPlacesClient() {
+        try {
+            placesClient = Places.createClient(this)
+            placesClientCreated.set(true)
+        } catch (ex: IllegalStateException) {
+            placesClientCreated.set(false)
+            logger.error("Places client has not been initialized yet.")
+        }
+    }
+
+
```

Comment:
Transferring the initialization to GooglePlacesService so that _Places.Client has a longer lifecycle than GooglePlacesManager and refrains from being instantiated multiple times (and creating multiple threads).